Lib/test/test_tools/test_msgfmt.py

```diff
@@ -121,10 +125,60 @@ def test_nonexistent_file(self):
         assert_python_failure(msgfmt, 'nonexistent.po')
 
 
+class MultiInputTest(unittest.TestCase):
+    """Tests for the issue https://github.com/python/cpython/issues/79516
+        msgfmt.py accepts multiple input files
```

Comment:
```suggestion
    """Tests for multiple input files
```

This is not a bug and we are not testing a reproducer. Also these are tests, not docs, no need to explain this.